interface/src/utils/keyCodes.js

```diff
@@ -0,0 +1,15 @@
+export const keyCodes = new Map([
```

Comment:
Dobrze rozumiem, że tutaj zapisujesz te bity w stringu? Może lepiej będzie zdefiniować inta, ale binarnie, jakoś tak: 
```
0b0000000
0b0000010
0b0000100
0b0000110
etc...
```
Sprawdź proszę jeszcze czy te wartości są prawidłowe, zgodne z README. Widzę, że ARROW_LEFT i ARROW_UP mają te same wartości

Reply:
Rzeczywiście zrobiłem błąd w wartościach, już poprawione. Co do sposobu zapisu, to można to zrobić na binarnych intach, tylko że je i tak trzeba przekonwertować do stringa, a wtedy js "ucina" zera z przodu, np. zamiast `00000101` dostajemy `101`. Pytanie jak backend będzie odczytywał te wartości. Bo jeśli brak zer z przodu nie będzie przeszkodą, to można tak zrobić. Ewentualność doklejania zer w pętli nie wydaje mi się za dobra.

Reply:
nie, brak zer nie będzie przeszkodą. Koniec końców, to mają być wartości liczbowe, tylko po prostu zdefiniowane bitowo. To będzie nic innego jak 0, 2, 4 etc. Na podstawie tych wartości mają być składane gotowe komunikaty, np ARROW_UP | KEY_DOWN, to binarny OR który mówi tyle, że została naciśnięta strzałka w górę. ARROW_UP | KEY_UP mówi, że została podniesiona i tak dalej.

Reply:
Zmieniłem je na format binarny, także teraz już wszystko powinno być OK